portfolio/src/main/webapp/script.js

```diff
@@ -40,36 +40,36 @@ async function getHello() {
  *Adds json messages to the message division
  */
 async function getMessage() {
-    const response = await fetch('/data');
-    const json = await response.json();
-    const message = "" + json.msg1 + " " + json.msg2 + " " + json.msg3;
-    document.getElementById('message-container').innerText = message;
+  const response = await fetch('/data');
+  const json = await response.json();
+  const message = "" + json.msg1 + " " + json.msg2 + " " + json.msg3;
```

Comment:
Why does the concatenation start with `"" +`?

Reply:
Changed the concatenation